rllib/examples/rl_module/random_rl_module.py

```diff
@@ -0,0 +1,36 @@
+from typing import Mapping, Any
+from ray.rllib.core.rl_module import RLModule
+import gymnasium as gym
+
+
+class RandomRLModule(RLModule):
```

Comment:
we should not merge this unimplemented. It should be quick to implement, so why not :)

Reply:
That would mean I have to create a unittest for it. The RandomPolicy won't even use these methods becuase it overrides compute_actions directly. That's why this PR just needs a placeholder for RandomRLModule. I would defer this to later.

Reply:
No it uses the old overriden compute_actions from the RandomPolicy.

Reply:
OH I see :).